config.json

```diff
@@ -1,58 +1,1349 @@
 {
-"general": {},
+    "advanced": {},
```

Comment:
We really should use the advanced as a sub-field to separate the common from the rare/advanced. The old devs really went overboard with settings.

Reply:
Yes, they did.